docroot/themes/humsci/humsci_basic/templates/field/pattern-view-field.html.twig

```diff
@@ -0,0 +1,55 @@
+{#
```

Comment:
We already have this template in `suhumsci/docroot/themes/humsci/humsci_basic/templates/views/pattern-view-field.html.twig`, so we should delete one of these. I believe they are the same, and I don't have a preference where it should belong.

Reply:
Oops, I figured this would go away when I rebased with your work. I'll delete.